test/unit/ngsiv2/provisioning/device-provisioning-configGroup-api_test.js

```diff
@@ -0,0 +1,1327 @@
+/*
```

Comment:
Is test/unit/ngsiv2/provisioning/device-provisioning-configGroup-api_test.js a "parallel" of some existing service-based test?

Reply:
Yes, It is parallel of test/unit/ngsiv2/provisioning/device-provisioning-api_test.js